php-frontend/src/test/java/org/sonar/php/symbols/ClassSymbolIndexTest.java

```diff
@@ -192,6 +194,45 @@ public void isSubTypeOf_with_cycle() {
     assertThat(a.isSubTypeOf(fqn("c"))).isEqualTo(Trilean.FALSE);
   }
 
+  @Test
+  public void get_method() {
+    ImmutableList<MethodSymbolData> methods = ImmutableList.of(
+      method("methodA"),
+      method("methodB")
+    );
+
+    ClassSymbolIndex symbols = createSymbols(
+      data("a", methods),
+      data("b")
+    );
+
+    ClassSymbol a = symbols.get(fqn("a"));
+    assertThat(a.methods()).hasSize(2);
+    assertThat(a.getMethod(fqn("methodA"))).isInstanceOf(MethodSymbolImpl.class);
+    assertThat(a.getMethod(fqn("random"))).isInstanceOf(UnknownMethodSymbol.class);
```

Comment:
I think that, most of the time, we don't want to assert the precise class and are more interested in testing `isUnknownSymbol()`.